components/file/src/main/java/org/trellisldp/file/FileBinaryService.java

```diff
@@ -228,7 +227,7 @@ public String generateIdentifier() {
     private File getFileFromIdentifier(final IRI identifier) {
         requireNonNull(identifier, "Identifier may not be null!");
         return of(identifier).map(IRI::getIRIString).filter(x -> x.startsWith("file:")).map(URI::create)
-            .map(URI::getSchemeSpecificPart).map(x -> stripStart(x, "/")).map(x -> new File(basePath, x))
+            .map(URI::getSchemeSpecificPart).map(x -> trimStart(x, "/")).map(x -> new File(basePath, x))
```

Comment:
I never noticed this-- does it work on Windows?

Reply:
Good question. It is certainly being tested on Windows, so my assumption has been that it works there.